src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,5 +60,11 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
+    
+    /**
+     * Returns true of the other name is very similar to this name.
+     * Two names are considered similar if ...
+     */
+     public boolean isSimilar(Name other) {}
```

Comment:
Good try, but you are a bit off from TDD. For example, a TDD session starts by adding a failing test, not by adding an empty method. Please take a look at the course handouts and suggested readings given in the assignment description to get a good understanding of TDD.